waspc/data/Generator/templates/react-app/src/operations/index.ts

```diff
@@ -0,0 +1,21 @@
+import api, { handleApiError } from '../api'
+import { HttpMethod } from '../types'
+
+export type OperationRoute = { method: HttpMethod, path: string }
+
+export async function callOperation(operationRoute: OperationRoute, args: any) {
+  if (operationRoute.method !== HttpMethod.Post) {
+    throw new Error(`Unsupported operation method: ${operationRoute.method}. Only POST is supported.`)
+  }
```

Comment:
You can catch this in the type system (especially if it's an internal function). Just change the argument type to `operationRoute: OperationRoute & { method: 'POST' }`.

Reply:
True, I will update the type, thnx! I mainly kept the runtime checks since the calling functions (our own) are still JS.